plugins/network/upnpc_

```diff
@@ -144,6 +147,14 @@ down.graph no
 up.negative down
 EOF
 			;;
+		"multi"|"upnpc")
+			echo "multigraph ${PLUGIN_NAME}"
+			config "traffic"
```

Comment:
Is the `traffic` really suitable as a *top level summary* of the other graphs?
Otherwise you may just want to graph all of them at the same level? (just replace the dot with an underscore)

Reply:
I think it is. It's a matter of preference though.

That said, we need at one top-level graph. We can't just have subpages.

Reply:
> That said, we need at one top-level graph. We can't just have subpages.

I wanted to propose to use underscores instead of dots in the multigraph names. In this case there would be no hierarchy and thus just a flat list of graphs at the top level.

Feel free to pick the most suitable approach for this set of information.

Reply:
Ah right, yeah. I never thought about it, actually.

That said, one of the reason I (ab)use multigraphs is to not have too many graphs on the front page, while still being able to drill down to more details if I want to. In this respect, I find the traffic most useful as a top-level graph, while the others are less so.

I think I'm going to leave it as is (: